src/main/java/com/easypost/utils/Fixture.java

```diff
@@ -0,0 +1,390 @@
+package com.easypost.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.internal.LinkedTreeMap;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.easypost.utils.Files.getSourceFileDirectory;
+import static com.easypost.utils.Files.readFile;
+
+/*
+ * This class needs to exist in the main package (rather than the test package)
+ * because this class needs to be referenced in the `HashMapSerializer`.
+ */
+public abstract class Fixture {
+
+    private static String readFixtureData() {
+        Path fixtureDataPath =
+                Paths.get(getSourceFileDirectory(), "examples/official/fixtures/client-library-fixtures.json");
+        return readFile(fixtureDataPath);
+    }
+
+    private static HashMap<String, Object> createFixture(String data) {
```

Comment:
Question: This is all very sad we need so many utils and can't just read from the file. Why do we need so many tools? Can we not do what we did in Go and coerce json blobs to classes?

Reply:
That's similar to what is happening here, in the sense that the large JSON string is being serialized into a giant map, then a sub-map is being extracted at a specific key.

I believe the difference comes where, in Go, all our functions are called with an object (an `Address`, a `Shipment`, etc.) used as the parameter, whereas in Java and C#, the functions take in maps of parameters. So, when it comes to deserializing the fixture JSON, we aren't using specific types for JSON properties, but rather generic map types.

We could potentially create a new model where the properties are different elements from the fixture JSON string (think the same way we make a `Shipment` object, with `Parcel` and `Address` properties, out of a normal JSON string), and then simply extract those properties to use as fixtures. I will try that now, but I'm unsure of the performance or clarity benefits it might bring (plus, read my comment above re: the benefits of the new HashMap serializer)

Reply:
I have changed how our fixtures are generated programmatically, going with a more explicit option that will is pro: more readable, con: more to maintain, unknown: more stable.

As a result, the fixtures exist again in the `test` namespace. I was also able to move some file-related utilities back to the `test` namespace, and remove fixture-related methods from the HashMap serializer.

I am still going to keep the new HashMap serializer for the benefits listed in my comment above, but in terms of loading the fixture JSON for unit tests, this does seem to be the better way to go (even if it undoes all my fancy trickery 🙃 )

Awaiting your feedback.